package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hal-json-vuex",
-  "version": "3.0.0-alpha.1",
+  "version": "3.0.0-alpha.2",
```

Comment:
Why this? We should only upgrade the version number when we actually publish a new verison, no?